test/integration/search/search_test.rb

```diff
@@ -473,6 +473,31 @@ def test_show_the_query
     assert parsed_response.fetch("elasticsearch_query")
   end
 
+  def test_dfid_can_search_by_every_facet
```

Comment:
Perhaps this can go in a separate file? Since it technically duplicates existing tests we could remove it once DFID stuff is live and stable.